user-script-register-v2/README.md

```diff
@@ -0,0 +1,19 @@
+# User script
+
+This extension demonstrates the [`browser.userScripts.register()`](https://wiki.developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/userScripts/RegisteredUserScript) API. This API enables an extension to register URL-matching scripts at runtime, scripts that run in isolated sandboxes.
+
+The extension includes an API script (`customUserScriptAPIs.js`) that provides stubs that enable a registered script to make use of `browser.storage.local`. 
```

Comment:
"API script" could helpfully link to https://wiki.developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/user_scripts. And these are not stubs: stubs do nothing. These functions wrap or expose the `storage.local` functions.